pkg/imgpkg/v1/describe.go

```diff
@@ -189,3 +196,30 @@ func (r *refWithDescription) describeBundleRec(visitedImgs map[string]refWithDes
 
 	return desc.bundle
 }
+
+func getImageLayersDigest(image string) []string {
+	layers := []string{}
+	parsedImgRef, err := regname.ParseReference(image, regname.WeakValidation)
+	if err != nil {
+		panic(fmt.Sprintf("Error: [%s] in parsing image: %s", err.Error(), image))
```

Comment:
Do we need to panic? Can we handle it more gracefully?

Reply:
`DescribeBundle` is not returning any error hence we have to use panic to show user error and exit from this. 
@joaopapereira is there any specific reason to not to return error from `DescribeBundle`  [here](https://github.com/carvel-dev/imgpkg/blob/develop/pkg/imgpkg/v1/describe.go#L109) ?

Reply:
In the past we did not need to, if we need to return errors now we should do it.